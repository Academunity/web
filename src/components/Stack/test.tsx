import { render } from 'utils/tests';

import { Stack } from '.';

describe('<Stack />', () => {
  it('should render correctly', () => {
    const { container } = render(
      <Stack>
        <div>Child 1</div>
        <div>Child 2</div>
        <div>Child 3</div>
      </Stack>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
