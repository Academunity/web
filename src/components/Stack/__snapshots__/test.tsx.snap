// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stack /> should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  class="c0"
  direction="column"
>
  <div>
    Child 1
  </div>
  <div>
    Child 2
  </div>
  <div>
    Child 3
  </div>
</div>
`;
